<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QWidget" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QPushButton" name="btnAssignPatch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Beet pachten</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtMemberName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="edtMemberNr">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>50</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MitgliedsNr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReleasePatch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pacht stornieren</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listPatches">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetListPatches">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Liste aktualisieren</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>120</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Admin</string>
   </property>
   <widget class="QPushButton" name="btnCreatePatches">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>create patches</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNewMember">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>new member</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnShowMemberPatches">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BeetListe anzeigen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
